test/Graphics/Texture.test.cpp

```diff
@@ -284,25 +252,20 @@ TEST_CASE("[Graphics] sf::Texture", runDisplayTests())
         constexpr std::uint8_t blue[]  = {0x00, 0x00, 0xFF, 0xFF};
         constexpr std::uint8_t green[] = {0x00, 0xFF, 0x00, 0xFF, 0x00, 0xFF, 0x00, 0xFF};
 
-        sf::Texture texture1;
-        REQUIRE(texture1.create(sf::Vector2u(1, 1)));
+        auto texture1 = sf::Texture::create(sf::Vector2u(1, 1), true).value();
         texture1.update(blue);
-        texture1.setSrgb(true);
         texture1.setSmooth(false);
         texture1.setRepeated(true);
 
-        sf::Texture texture2;
-        REQUIRE(texture2.create(sf::Vector2u(2, 1)));
+        auto texture2 = sf::Texture::create(sf::Vector2u(2, 1), false).value();
         texture2.update(green);
-        texture2.setSrgb(false);
         texture2.setSmooth(true);
         texture2.setRepeated(false);
 
         sf::swap(texture1, texture2);
         CHECK_FALSE(texture1.isSrgb());
         CHECK(texture1.isSmooth());
         CHECK_FALSE(texture1.isRepeated());
-        CHECK(texture2.isSrgb());
```

Comment:
Something to know about `setSrgb()` prior to this PR is that it must be called before `create()` for it to work. In this case it was called after `create()` meaning the texture didn't actually attempt to enable SRGB.

My API refactor means this parameter must be provided upon creation. As it turns out this feature doesn't work on Windows OpenGL ES so the `create()` function automatically disables it and emits a warning. That means we can't test for `isSrgb` being `true` since on that one config it's false.

Reply:
A comment explaining this in the code would be valuable so it doesn't look like testing it was just overlooked.